drivers/windows/file_access_windows.cpp

```diff
@@ -369,6 +370,11 @@ Error FileAccessWindows::get_error() const {
 	return last_error;
 }
 
+Error FileAccessWindows::resize(int64_t p_length) {
+	ERR_FAIL_NULL_V_MSG(f, FAILED, "File must be opened before use.");
+	return (_chsize_s(_fileno(f), p_length) == 0) ? OK : FAILED;
```

Comment:
The Windows API defines the following error returns:
> EACCES if the specified file is locked against access
> EBADF if the specified file is read-only or the descriptor is invalid
> ENOSPC if no space is left on the device
> EINVAL if size is less than zero

IMO we should start being better about giving developers error information, to they can handle those appropriately. "Game save failed because no space is left on device" and "Game save failed due to lack of permissions on..." are both more informative and actionable than "Game save failed" for players and devs/testers during development alike.

We could use:
- `ERR_FILE_CANT_OPEN` for `EACCESS`/`EBADF`
- `ERR_OUT_OF_MEMORY` for `ENOSPC`
- `ERR_INVALID_PARAMETER` for `EINVAL`
...in addition to `OK`/`FAILED`.